mechanical/differential_pivot/README.md

```diff
@@ -31,8 +38,143 @@ The Differential Pivot helps to offload forces from one side of the rover to the
 ### Optional
 
 ## Dependencies on Other Modules:
-  * None
 
 ## Mechanical Interface/Attachments to Rover:
   * 1/2 inch clamping hub to 1/2 inch aluminum rod on the body
   * 1/2 inch aluminum rod to 1/2 inch clamping hub on rocker-bogie arms
+
+## Machining/Fabrication
+
+1. Aluminum Rods: Cutting
```

Comment:
In the other readmes (and the template) we use headings instead of numbered lists.

So `### 3.1 Aluminum Rods: Cutting`